entities/weapons/weapon_cs_base2/sv_commands.lua

```diff
@@ -56,6 +56,9 @@ function meta:dropDRPWeapon(weapon)
 end
 
 local function DropWeapon(ply)
+    local curTime = CurTime()
+    if (ply.DelayDropWeapon or 0) >= curTime then return end --Fix Dupe Weapon
```

Comment:
This function returns the empty string. This empty string represents the replacement chat value.